sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateRunDao.java

```diff
@@ -37,7 +37,7 @@
 public class HibernateRunDao implements RunStore, HibernatePaginatedDataSource<Run> {
 
   private static final List<String> STANDARD_ALIASES = Arrays.asList("lastModifier", "creator",
-      "sequencerReference", "sequencerReference.platform");
+      "sequencer", "sequencer.platform");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "sequencer" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)